composites/AlgoliaSearch/AlgoliaSearcher.js

```diff
@@ -336,48 +336,27 @@ class AlgoliaSearcher extends React.Component {
 	 * @returns {ReactElement|string} Returns a header if there are search results. Otherwise returns an empty string.
```

Comment:
technically, "Otherwise returns an empty string" is not correct, because it returns `null`.